tests/acceptance/helpers/sharingHelper.js

```diff
@@ -131,8 +165,33 @@ module.exports = {
           throw Error('Could not get shares. Message: ' + sharesResult.ocs.meta.message)
         }
       })
-
-    return lastShareToken
+    return lastShare
+  },
+  deleteLastPublicShare: async function (sharer, shareId) {
+    const headers = httpHelper.createAuthHeader(sharer)
+    const apiURL = client.globals.backend_url + '/ocs/v2.php/apps/files_sharing/api/v1/shares/' + shareId
+    await fetch(apiURL, { method: 'DELETE', headers: headers })
+  },
+  assertUserLastPublicShareDetails: async function (linkCreator, expectedDetails) {
+    const lastShare = await this.fetchLastPublicLinkShare(linkCreator)
+    if (lastShare !== '') {
```

Comment:
lastShare is not string. Since `fetchLastPublicLinkShare()` not returns share object and not shareToken it given object.